### PR TITLE
UPSTREAM: <carry>: Update OWNERS file

### PR DESCRIPTION
**Pull request by bharath-b-rh**
Update OWNERS with OpenShift cert-manager maintainers.

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,6 +1,6 @@
-approvers:
-- cm-maintainers
 reviewers:
-- cm-maintainers
-- spectralhiss
-- thatsmrtalbot
+  - TrilokGeer
```
Comment (TrilokGeer) [inline OWNERS:2]: The owners and maintainers from upstream list should be retained.

Comment (TrilokGeer) [reply OWNERS:2]: This can be maintained as similar to cert-manager downstream and possibly independent of carry PRs.

Comment (bharath-b-rh) [reply OWNERS:2]: Thank you! closing this PR.

**Pull request closed without merging**